python/src/gen3authz/client/arborist/base.py

```diff
@@ -114,11 +109,13 @@ def make_context(self, kwargs):
         return EnvContext(self._get_stack(), kwargs)
 
 
-class ArboristClient(AuthzClient):
```

Comment:
this is a breaking change for the lib, right? or clients can use the new async version without changing anything on their end?

Reply:
As mentioned in the PR description, this is not a breaking change. `gen3authz.client.arborist.client` provides the same blocking API as it used to be, existing clients are not using the async version (`gen3authz.client.arborist.async_client`), so they won't need to change anything.